packages/core/src/components/modal/modal.scss

```diff
@@ -147,7 +147,7 @@ $modals: (
     .body {
       font: var(--tds-body-01);
       letter-spacing: var(--tds-body-01-ls);
-      padding-bottom: 96px;
+      padding-bottom: 35vh;
```

Comment:
Be careful when using relative values, we might end up in a scenario like this as it is dependent on screen height.





![Screenshot 2024-04-02 at 16 59 46](https://github.com/scania-digital-design-system/tegel/assets/17004063/195a614c-ac50-4400-95ae-d3234a9a7733)


See issue is in how slots affect each other. If title goes to two or more lines, it affect body. Body is calculated with calc which seems to get confused if Title gets to push it. Please do play with flex, flex basis and flex shrink work, maybe it can help out here.

Check this article and see if we can use something like that on Modal component here: https://css-tricks.com/understanding-flex-grow-flex-shrink-and-flex-basis/
Maybe it is possible to use tricks form here, but with flex-orientation: column. 
Do play with it, it is a nice challenge :)

Reply:
Realized about an hour after I went home that this fix wouldn't work, gotta love those "wait a minute" moments in development 😅 I'll take a look at flex and see what I can do!

Reply:
Good suggestion! Yes, most likely then the modal would be set to display: flex; so:
`
.tds-modal {
  display: flex;
  flex-direction: column;
  `
  kinda.